crates/sui-node/src/lib.rs

```diff
@@ -163,15 +166,23 @@ impl SuiNode {
             LogCheckpointOutput::boxed()
         };
 
+        let (certified_checkpoint_output, init_state_sync_handle) =
+            sui_core::checkpoints::SendCheckpointToStateSync::new();
```

Comment:
nit: import?

Reply:
If something is only ever used once in the file i've generally opted towards not importing it in order to avoid import clutter. although maybe i shouldn't care about that